keyboards/varys/keymaps/default/keymap.c

```diff
@@ -0,0 +1,266 @@
+    ),
+    [_RAISE] = LAYOUT(
+    _______, _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+                               _______, _______, _______, _______, _______,  _______, _______, _______
+    ),
+    [_MISC] = LAYOUT(
+    _______, _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______,                   _______, _______, _______, _______, _______, _______,
+    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
+                               _______, _______, _______, _______, _______,  _______, _______, _______
+    )
+};
+
+
+#ifdef RGB_MATRIX_ENABLE
+/* LED 
+ *  {0,0}                                                                                             {224,0}
+ * ,-----------------------------------------.                    ,-----------------------------------------.
+ * |  5   |   4  |   3  |   2  |   1  |   0  |                    |  28  |  29  |  30  |  31  |  32  |  33  |
+ * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+ * |  11  |  10  |   9  |   8  |   7  |   6  |                    |  34  |  35  |  36  |  37  |  38  |  39  |
+ * |------+------+------+------+------+------|                    |------+------+------+------+------+------|
+ * |  17  |  16  |  15  |  14  |  13  |  12  |-------.    ,-------|  40  |  41  |  42  |  43  |  44  |  45  |
+ * |------+------+------+------+------+------| NO LED|    |NO LED |------+------+------+------+------+------|
+ * |  23  |  22  |  21  |  20  |  19  |  18  |-------|    |-------|  46  |  47  |  48  |  49  |  50  |  51  |
+ * `-----------------------------------------/       /     \      \-----------------------------------------'
+ *                   | 27   |26    |25   |  /24     /       \52    \  |53    |54     |55    |
+ *                   |      |      |     | /       /         \      \ |      |      |      |
+ *                   `----------------------------'           '------''--------------------'         
+ *  {64,0}                                                                                           {224,64}      
+ */
+const is31fl3737_led_t PROGMEM g_is31fl3737_leds[IS31FL3737_LED_COUNT] = {
```

Comment:
This should not be in a keymap.  This should be in `varys.c`, alongside keyboard.json.